#Request
---
#Response
float32 distance
float32 xVelAverage
float32 zVelAverage





